packages/node/test/onunhandledrejection.test.ts

```diff
@@ -1,15 +1,29 @@
-import { Hub, init, Integrations, Scope } from '../src';
+import { Scope } from '@sentry/core';
+import { Hub } from '@sentry/hub';
 
-const dsn = 'https://53039209a22b4ec1bcc296a3c9fdecd6@sentry.io/4291';
+import { OnUnhandledRejection } from '../src/integrations/onunhandledrejection';
+
+jest.mock('@sentry/hub', () => {
+  // we just want to short-circuit it, so dont worry about types
```

Comment:
I understand the rest of the change, don't fully understand this.

Why do we want to short-circuit?

Reply:
Because `setupOnce` is checking whether an integration is installed, which in the case of these tests should always be the case. I think this was the reason for flakyness, because `hub` is attached to the global object.